src/ClassicUO.Client/LegionScripting/API.py

```
@@ -24,293 +25,1385 @@
 def Pathfind(x: int, y: int, z: int = 1337, distance: int = 0):
+    """
+     Attempt to pathfind to a location.  This will fail with large distances.
+     Example:
+     ```py
+     API.Pathfind(1414, 1515)
+     ```
+    
+    """
     pass
 
 def Pathfind(entity: int, distance: int = 0):
```

Ruff (0.11.9)

Redefinition of unused `Pathfind` from line 515

(F811)